app/services/reject_by_default_feedback.rb

```diff
@@ -19,17 +19,42 @@ def save
         application_choice.save!
       end
 
-      CandidateMailer.feedback_received_for_application_rejected_by_default(application_choice).deliver_later
+      show_apply_again_guidance = unsuccessful_application_choices? && not_applied_again?
+
+      CandidateMailer.feedback_received_for_application_rejected_by_default(application_choice, show_apply_again_guidance).deliver_later
       notify_slack
     end
   end
 
   def notify_slack
     provider_name = application_choice.current_course.provider.name
-    candidate_name = application_choice.application_form.first_name
+    candidate_name = application_form.first_name
     message = ":telephone_receiver: #{provider_name} has sent feedback for #{candidate_name}’s RBD application"
-    url = Rails.application.routes.url_helpers.support_interface_application_form_url(application_choice.application_form)
+    url = Rails.application.routes.url_helpers.support_interface_application_form_url(application_form)
 
     SlackNotificationWorker.perform_async(message, url)
   end
+
+  def not_applied_again?
+    ApplicationForm
+      .where(
+        candidate_id: candidate.id,
+        recruitment_cycle_year: RecruitmentCycle.current_year,
+        previous_application_form_id: application_form.id,
+      )
+      .empty?
```

Comment:
Ah, I didn't realise that a `previous_application_form_id` was set when applying again.

Looks like there’s an [inverse relationship](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/models/application_form.rb#L21) set on the model too. Could we use that here?

```suggestion
  def not_applied_again?
    application_form.subsequent_application_form.nil?
```

Reply:
💯